bigquery_etl/util/udf_functions.py

```diff
@@ -0,0 +1,29 @@
+"""Utility functions related to UDF parsing and documentation."""
+
+import re
+
+
+def find_input(sql_text):
+    """Extract input parameters from UDF sql code."""
+    sql_text = re.sub(r"\s{2,}|\\n", " ", sql_text)
+    return (
+        re.search(r"FUNCTION [a-z,0-9,_,.]+\s*\(\s*(.+?)\s*\)", sql_text)
+        .groups()[0]
+        .strip()
```

Comment:
This shouldn't be necessary, given that we are already stripping whitespace characters with the regex.

Reply:
we only strip double white spaces ... I added it for `test_find_input_multiline_input` where there is a line break before the closing parathesis

Reply:
oh no you are right ... it doesn't break any more ... thanks for catching that .. must have been a leftover from a previous iteration